ui/lesson/ScriptingChallenge/index.tsx

```diff
@@ -33,20 +33,22 @@ export default function ScriptingChallenge({
   config,
   successMessage,
   onSelectLanguage,
+  hiddenRange,
 }: {
   children?: React.ReactNode
   lang: string
   lessonKey: string
   config: EditorConfig
   successMessage: string
   onSelectLanguage: (language: string) => void
+  hiddenRange: number[]
```

Comment:
this needs to be optional to build properly. I did change how the hiddenRange works so hopefully this won't be needed